resources/rust/main.rs

```diff
@@ -1,9 +1,14 @@
 //! A blinky example for Tessel
 
+// This enables the system allocator in your script.
+// This is required to run Rust binaries on Tessel.
+#![feature(alloc_system)]
+extern crate alloc_system;
```

Comment:
@tcr just FYI, this breaks remote code deployment because the cross compilation server is running Rust `v1.10.0` but not nightly.